### PR TITLE
Add livearea

### PR DESCRIPTION
**Pull request by TakuyaKousaka**
At least there's something now

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,4 +47,8 @@ vita_create_vpk(
 	VERSION ${VITA_VERSION}
 	NAME ${VITA_APP_NAME}
 	FILE bin/monorale.bin monorale.bin	
+		 src/sce_sys/icon0.png sce_sys/icon0.png
```
Comment (knight-ryu12) [inline CMakeLists.txt]: without FILE, CMAKE won't recognize.

**Review by knight-ryu12 — CHANGES_REQUESTED** [review]
This will need change in across all changes in CMakeLists

File: CMakeLists.txt (hunk 1 above)
Comment (TakuyaKousaka) [reply CMakeLists.txt]: It compiled and included all of the files on my end though?

Comment (knight-ryu12) [reply CMakeLists.txt]: but you should follow coding basis.

Comment (TakuyaKousaka) [reply CMakeLists.txt]: Alright

**Comment by knight-ryu12** [comment]
Looks good to me.

**Pull request merged**